app/Repositories/PostRepository.php

```diff
@@ -120,7 +120,20 @@ public function create(array $data, $signupId)
     public function update($signup, $data)
     {
         if (array_key_exists('updated_at', $data)) {
-            $signup->fill(array_only($data, ['quantity', 'quantity_pending', 'why_participated', 'updated_at']));
+            // Only update if the key is set (is not null).
+            $arrayKeysToUpdate = ['why_participated', 'quantity'];
+
+            foreach ($arrayKeysToUpdate as $key => $value) {
+                if (! $data[$value]) {
+                    $key = array_search($value, $arrayKeysToUpdate);
+                    unset($arrayKeysToUpdate[$key]);
+                }
+            }
+
+            array_push($arrayKeysToUpdate, 'updated_at');
+            array_push($arrayKeysToUpdate, 'quantity_pending');
```

Comment:
I don't think anyone is ever sending us these values? I think updated_at was for the migration (Laravel handles that now) and I don't think anyone sends over quantity_pending.

Reply:
Oh I see now that this is in the `updated_at` section. Nevermind!!!